### ISSUE TITLE
VSD loss NaN

### ISSUE THREAD

**Issue by zzzzzero** [cswry/OSEDiff#44]
Thank you for open-sourcing the training code.
 When I used the training code, I noticed that when using VSD loss with lambda_vsd set to 1, the loss becomes NaN after about 2000 steps. However, when I don't use this part of the loss and set lambda_vsd to 0, the loss and results are normal. I wanted to ask if you encountered this issue during the training process.

**Comment by Kiteretsu77** [comment]
I face the same issue.

**Comment by Kiteretsu77** [comment]
I changed the version back to original defined SD2.1-Base (from SD2.1 without Base) and made some small modifications (like making lr smaller), it becomes good now! Thank you very much!

**Comment by zzzzzero** [comment]
> I changed the version back to original defined SD2.1-Base (from SD2.1 without Base) and made some small modifications (like making lr smaller), it becomes good now! Thank you very much!

I made the same mistake as you, using sd21 instead of sd21-base. I suspect that sd21 does not generate 512x512 images very well, which caused the issues with VSD loss. I'm now using sd21-base, and the loss and results seem normal. Thank you for your timely responses!
